internal/tiltfile/tiltfile_test.go

```diff
@@ -2866,7 +2866,7 @@ k8s_resource('foo', port_forwards=8000)
 	f.loadErrString("k8s_resource already called for foo")
 }
 
-func TestK8sResourceEmptyWorkloadSpecifier(t *testing.T) {
+func TestK8sResourceEmptyWorkloadSpecifierAndNoObjects(t *testing.T) {
```

Comment:
super nbd, if you feel inspired i might reorder the tests so that all the k8s_resource stuff is together instead of being split by workload_to_resource tests (there's a chunk up here and a chunk around ln4449) but 🤷